### PR TITLE
Python update

### PR DESCRIPTION
**Pull request by jaxxzer**

### CODE REVIEW

File: src/protocol/generate-python.py [hunk 1]
```diff
@@ -76,11 +77,20 @@ def is_vector(payload):
 if not os.path.exists(output_path):
     os.makedirs(output_path)
 
+try:
+    os.chmod(os.path.join(output_path, "PingMessage.py"), stat.S_IWUSR | stat.S_IWGRP | stat.S_IWOTH)
```
Comment (patrickelectric) [inline src/protocol/generate-python.py]: Create a variable that is `var = tat.S_IWUSR | stat.S_IWGRP | stat.S_IWOTH` and what this variables together does.

File: src/protocol/generate-python.py [hunk 2]
```diff
@@ -76,11 +77,20 @@ def is_vector(payload):
 if not os.path.exists(output_path):
     os.makedirs(output_path)
 
+try:
```
Comment (patrickelectric) [inline src/protocol/generate-python.py]: If I for the idea right, this is not working, the script is not able to change the permissions here. 
```
 patrick@dell  /mnt/hd/git/blue/ping-protocol-cpp/src/protocol   python-update  ./generate-python.py                                                   
 patrick@dell  /mnt/hd/git/blue/ping-protocol-cpp/src/protocol   python-update  ./generate-python.py
Traceback (most recent call last):
  File "./generate-python.py", line 93, in <module>
    f = open(os.path.join(output_path, "Ping1D.py"), "w")
PermissionError: [Errno 13] Permission denied: '/mnt/hd/git/blue/ping-protocol-cpp/src/protocol/python/Ping1D.py'

```

File: src/protocol/generate-python.py [hunk 3]
```diff
@@ -76,11 +77,20 @@ def is_vector(payload):
...
+    os.chmod(os.path.join(output_path, "PingMessage.py"), stat.S_IWUSR | stat.S_IWGRP | stat.S_IWOTH)
+except FileNotFoundError as e:
+    pass
 f = open(os.path.join(output_path, "PingMessage.py"), "w")
 f.write(j2_env.get_template("PingMessage.py.in").render(jsondata))
 f.close()
+os.chmod(os.path.join(output_path, "PingMessage.py"), stat.S_IRUSR | stat.S_IRGRP | stat.S_IROTH)
```
Comment (patrickelectric) [inline src/protocol/generate-python.py]: The same thing for this other 3.

File: src/protocol/generate-python.py [hunk 4]
```diff
@@ -76,11 +77,20 @@ def is_vector(payload):
...
 
+try:
+    os.chmod(os.path.join(output_path, "PingMessage.py"), stat.S_IWUSR | stat.S_IWGRP | stat.S_IWOTH)
```
Comment (patrickelectric) [inline src/protocol/generate-python.py]: This should be `"Ping1D.py"`

**Comment by patrickelectric** [comment]
I'm also not having permitsion to read that files..
```
Traceback (most recent call last):
  File "./generate-json.py", line 18, in <module>
    import PingMessage as Message
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 859, in get_code
  File "<frozen importlib._bootstrap_external>", line 916, in get_data
PermissionError: [Errno 13] Permission denied: '../src/protocol/python/PingMessage.py'
```

**Comment by patrickelectric** [comment]
I think that we should not make the files read only, this only restringes our workflow and the user as well. Don't see big advantages on it.

**Comment by jaxxzer** [comment]
closed in favor of #103

**Pull request closed without merging**